src/app/users/users.module.ts

```diff
@@ -0,0 +1,30 @@
+import { NgModule } from '@angular/core';
+import { UserDeleteConfirm, UsersComponent } from './users.component';
+import { UserEditComponent } from './user-edit/user-edit.component';
+import { RouterModule } from '@angular/router';
+import { UsersRoutingModule } from './users-routing.module';
+import { FormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { MatListModule } from '@angular/material/list';
```

Comment:
create a new module called SharedModule.ts
move all the Material dependencies there.

then use the shared module Here.

that way you can re use that on many other modules. that use Material dep.

Search for that classic SharedModule examples